SPARQLWrapper/Wrapper.py

```diff
@@ -1310,8 +1327,12 @@ def __get_results_width(self, results, minWidth=2):
                 index += 1
         return width
 
-    def __get_prettyprint_string_sparql_var_result(self, result):
+    def __get_prettyprint_string_sparql_var_result(self, result: Dict[str, str]) -> str:
+        if "value" not in result:
+            return ""
```

Comment:
Is this an expected condition/eventuality? If not it may be best to just leave it as it was so it raises an error.

Reply:
90cff98